src/Entity/Game.php

```diff
@@ -20,10 +21,35 @@ class Game
     private $id;
 
     /**
-     * @ORM\OneToMany(targetEntity="App\Entity\Ship", mappedBy="game")
+     * @ORM\Column(type="string", length=255)
+     */
+    private $hash;
+
+    /**
+     * @ORM\OneToOne(targetEntity="App\Entity\Player", cascade={"persist"})
```

Comment:
There's too much `cascade={persist}` . You don't need it on all the entity attributes... You generally never change the player1 from the game itself

Reply:
The players are created when we persist the new game.

Reply:
:+1: